Aws api module add sts (#17447)
demisto/content: Packs/AWS-ACM/ReleaseNotes/1_1_8.md

#### Integrations
##### AWS - ACM

> - AWSClient Class in ApiModules/AWSApiModule now supports passing an AWS STS session token as a parameter
> - Allows for third-party authorization software such as Hashicorp Vault to provide login credentials to use for building a client session

Reviewer: ```suggestion
- AWSClient class now supports passing an AWS STS session token as a parameter. This allows for third-party authorization software to provide login credentials to use for building a client session.
```
